style.css

```diff
@@ -3310,7 +3315,7 @@ article.panel-placeholder {
 		width: auto;
 	}
 
-	/* Gallery Columns */
+	/* Gallery Columns */
```

Comment:
Is this change a switch from a tab to a space? i.e. I hope it's not the other way around from a space to a tab.